dali/pipeline/operators/operator.h

```diff
@@ -256,6 +256,9 @@ void CollectPointersForExecution(size_t batch_size,
                                  const TensorList<GPUBackend> &input, vector<const uint8 *> *inPtrs,
                                  TensorList<GPUBackend> &output, vector<uint8 *> *outPtrs);
 
+using OpPtr = std::unique_ptr<OperatorBase>;
```

Comment:
I wonder if declaring `OpPtr` here is really a good idea. It's a very short name, that can easily collide with something.

Reply:
Agree, I'll remove it